app/config.py

```diff
@@ -193,5 +203,8 @@ def __getitem__(self, name):
             return self.global_config[name]
         return None
 
+    def get(self, name, default=None):
+        return self[name] or default
```

Comment:
I believe this returns `default` when the value is 0 or ''. Can you return `default` only when the value is None?

It may also be helpful to document why you need to explicitly define this method because it is also provided by UserDict.DictMixin (I guess because `__getitem__` doesn't throw an exception for non-existing keys?).

Reply:
Done. And yeah, `DictMixin.get` returns the default only if an exception is raised (it was kinda hard to find documentation of its behavior, but I tested it out).